generated-platform-project/quarkus-universe/bom/pom.xml

```diff
@@ -21777,97 +21566,97 @@
       <dependency>
         <groupId>org.eclipse.jetty.http2</groupId>
         <artifactId>http2-client</artifactId>
-        <version>9.4.45.v20220203</version>
+        <version>9.4.46.v20220331</version>
```

Comment:
@ppalaga are Jetty dependencies actually used by CEQ?

Reply:
I think we have `camel-quarkus-support-jetty` which is used by `camel-quarkus-salesforce` and some integration tests.